src/orquestra/sdk/_base/serde.py

```diff
@@ -34,11 +35,28 @@ def _chunkify(s: str) -> t.List[str]:
 def _encoded_pickle_chunks(object: t.Any) -> t.List[str]:
     return _chunkify(
         codecs.encode(
-            dill.dumps(object, protocol=PICKLE_PROTOCOL, recurse=True), ENCODING
+            cloudpickle.dumps(object, protocol=PICKLE_PROTOCOL), ENCODING
         ).decode()
     )
 
 
+@contextmanager
+def registered_module(module):
```

Comment:
**suggestion**: could you add a docstring that states why we need it/when it's good to use it?

Reply:
Wdyt about 0edb446?